SPECS/ocaml-gnt.spec

```diff
@@ -15,6 +15,7 @@ BuildRequires:  ocaml-cstruct-devel
 BuildRequires:  ocaml-io-page-devel
 BuildRequires:  ocaml-lwt-devel
 BuildRequires:  ocaml-cmdliner-devel
+BuildRequires:  ocaml-mirage-profile-devel
 BuildRequires:  xen-devel
 BuildRequires:  xen-dom0-libs-devel
 BuildRequires:  xen-dom0-libs
```

Comment:
This line should not be necessary - xen-dom0-lib-devel should require xen-dom0-libs.